index.html

```diff
@@ -370,5 +373,66 @@ <h3>LET’S CONNECT</h3>
       </section>
     </main>
     <!-- end sectionsix -->
+    <!-- start footer -->
+    <div class="white">
+      <footer class="footer" id="footer">
+        <div class="container">
+          <div class="footer__container__parfoot">
+            <h3>Contact me</h3>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <form action="https://formspree.io/f/mvoyalje" method="post">
+              <div class="form__control">
+                <input
+                  type="text"
+                  placeholder="Name"
+                  class="name"
+                  id="name"
+                  maxlength="30"
+                  required
+                  name="name"
+                />
+              </div>
+              <div class="form__control">
+                <input
+                  type="email"
+                  placeholder="name@example.com"
+                  class="email"
+                  id="email"
+                  name="email"
+                  required
+                />
+              </div>
+              <div class="form__control">
+                <textarea
+                  id=""
+                  cols="30"
+                  rows="10"
+                  class="textarea"
+                  id="textarea"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  name="textarea"
+                  required
+                ></textarea>
+              </div>
+              <div class="form__control">
+                <input
+                  type="submit"
+                  placeholder=""
+                  class="submit"
+                  id="submit"
+                  value="Get in touch"
+                />
+              </div>
```

Comment:
- You did a great job by adding the submit button and it working as expected but is just visible in the desktop version. Kindle makes sure the button for submitting the form is also visible in the mobile version as required by the project. check below your design compared to the Figma design 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/192584805-a695284a-40d2-42b3-9b20-2a5a042d5dde.png)

## Figma design

![image](https://user-images.githubusercontent.com/95297251/192585151-9d3e4129-a6e3-4e33-a614-e04e5997c388.png)

Reply:
- Thanks  @MaganaAsiati 🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊